### ISSUE TITLE
Private keys

### ISSUE THREAD

**Issue by g1st** [chingu-voyages/v8-bears-team-14#14]
Need to sort out .env file and private keys. Current keys are:
* MAIL_API_PASS
* MAIL_API_USER
* MAIL_HOST
* MAIL_PORT
* STRIPE_PUBLISHABLE_KEY
* STRIPE_SECRET_KEY

It's ok for me to share mine or you getting your own? Mail client used is [sendgrid](http://www.sendgrid.com). Free tier allows 100 free mails per day.

**Comment by nikrb** [comment]
How about we fake stripe and email for running locally? That way we can all develop and then add the real detail in when deployed?

**Comment by g1st** [comment]
Just to be sure. You mean to register to those services with newly created fake email? If so, I can do that!

**Comment by nikrb** [comment]
@g1st no. I mean we put fake keys/urls etc in .env and rig up a backend to respond as it they were those services - without actually using the real services.

**Comment by nikrb** [comment]
The test api for stripe sounds interesting. I think I'd still prefer an instant response when working locally, esp on frontend. thoughts?

**Comment by g1st** [comment]
I suggest to go this [way](https://github.com/chingu-voyages/v8-bears-team-14/issues/12#issuecomment-477067526). But if you still prefer instant fake response for development feel free to fake it and I would happily use that for dev.
